src/compiler/crystal/repl/interpreter.cr

```diff
@@ -1161,7 +1161,8 @@ class Crystal::Repl::Interpreter
       while @pry
         # TODO: supoort multi-line expressions
 
-        line = Readline.readline("pry> ", add_history: true)
+        print "pry> "
```

Comment:
This is a super small comment.  I can even try to change this if we come up with a different name. do we want to use the name pry?  That seems like it might come with other expectations from the Ruby community.  What about just having the prompt be `repl`

Reply:
That's the kind of discussions I'd like to avoid in this PR. Please let's talk about that later. Nothing is set in stone. We just need to get this merged into master to start working on these details.

Reply:
totally makes sense.  I was just trying to be helpful.